nepalingo-web/src/hooks/StreakContext.tsx

```diff
@@ -31,7 +31,7 @@ export const StreakProvider = ({ children }: { children: ReactNode }) => {
         .from("user_daily_streaks")
         .select("*")
         .eq("user_id", user.id)
-        .single();
+        .maybeSingle();
```

Comment:
Without this, `single()` would raise an error for new users because there would be no rows in the db for them.